wp-dependency-installer.php

```diff
@@ -369,6 +377,19 @@ public function is_active( $slug ) {
 			return is_plugin_active( $slug );
 		}
 
+		/**
+		 * Check current admin screen id.
+		 *
+		 * @param  string $screen_id screen to check.
+		 *
+		 * @return boolean check we are displaying selected screen.
+		 */
+		public function is_admin_screen( $screen_id ) {
+			global $current_screen;
+
+			return isset( $current_screen ) && isset( $current_screen->id ) && $screen_id === $current_screen->id;
+		}
+
```

Comment:
this function can also be used to limit the display of nag notices on unwanted admin pages